.github/workflows/build.yml

```
@@ -17,16 +17,18 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
-                symfony: ["5.4.*", "6.0.*", "6.1.*"]
+                php: ["8.1", "8.2", "8.3"]
+                symfony: ["5.4.*", "6.4.*", "7.0.*"]
```

```suggestion
                symfony: ["^5.4", "^6.4", "^7.0"]
```
and all the places below